demo/README.md

```diff
@@ -25,3 +25,14 @@ Commonly, you will want to bind-mount your host's `/etc/ceph` into the container
 For example:
 
 `docker run -d --net=host -v /etc/ceph:/etc/ceph -e MON_IP=192.168.0.20 -e CEPH_NETWORK=192.168.0.0/24 ceph/demo`
+
+
+Tip
+---
+
+**Note**: it can be dangerous
```

Comment:
Hi @Ulexus 

I'm not natural english speaker, could you help me ?
Remount sounds danger, I'm not sure how tell it.

thanks